### DISCUSSION TITLE
Wezterm ignore WEZTERM_CONFIG_FILE environement variable

### DISCUSSION THREAD

**Discussion by barmic** [wezterm/wezterm#4819] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

I use X11 on debian 12.4 with i3 window manager

### WezTerm version

20240119-085337-05eadb7c

### Ask your question!

I want to put my wezterm config in `/etc` so I put my file in `/etc/wezterm.lua` and I export a `WEZTERM_CONFIG_FILE=/etc/wezterm.lua` in my zshrc file (and I can see it with `env | awk -F= '$1=="WEZTERM_CONFIG_FILE"'`).

```
.rw-r--r-- 1,7k root 20 janv. 18:50 /etc/wezterm.lua
```

But the variable seem's ignored. When I run `wezterm --config-file /etc/wezterm.lua`, it's correctly applied.

I have only found an old [discussion on MacOS on this subject](https://github.com/wez/wezterm/discussions/635).

Anybody have an idea about why this don't work? How can I debug this (I don't found cli parma to debug it)?

**Comment by wez** [comment, marked as answer]
You can set `WEZTERM_LOG=config=trace,info` in the environment to debug the config crate

**Reply by barmic** [reply]
Thanks, it is a problem with setting the environment variable.
